mixer/testdata/config/rbac.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: authorization
+metadata:
+  name: requestcontext
+  namespace: istio-system
+spec:
+  subject:
+    user: request.auth.principal | ""
+    groups: request.auth.principal | ""
+    properties:
+      service: source.service | ""
+      namespace: source.namespace | ""
+  action:
+    namespace: destination.namespace | ""
+    service: destination.service | ""
+    method: request.method | ""
+    path: request.path | ""
+    properties:
+      version: request.headers["version"] | ""
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: rbac
+metadata:
+ name: rbachandler
```

Comment:
super nit: just call this `handler`, so the name becomes `handler.rbac` (and avoids stutter).

Reply:
Done.